Benchmarking/GC_stub/RnD1system_benchmarking_analysis_GCstub.py

```diff
@@ -0,0 +1,285 @@
+"""
+Analyse EC-MS data from a benchmarking exp. as recorded on a GC electrode.
+
+Created on Wed Feb  2 09:30:01 2022.
+
+@author: Anna Winiwarter
+Spectro Inlets A/S
+contact: support@spectroinlets.com
+"""
+
+from pathlib import Path
+
+import numpy as np
+import ixdat
+from matplotlib import pyplot as plt
+from ixdat.techniques.ec import ECMeasurement
+
+# resolve parent directory
+THIS_DIR = Path(__file__).parent.resolve()
+
+
+# -----------------------EDIT SETTINGS HERE----------------------------------
+# Settings for choosing which part of the script is executed:
+DATA_SOURCE = "raw"
+# DATA_SOURCE can be "raw" (for importing EC and MS data from Zilien .tsv file),
+# or "ixdat" (for importing ixdat .csv files as generated when using setting "raw")
+# in either case, for plotting the CV only, the biologic CVA file is imported
+# automatically
+WHICH_PART = "plot_CVs"
+# WHICH_PART can be "plot_CVs", "plot+fit_HER", "plot+fit_gas_exchange"
+
+SAVE_FIGURES = True
+# SAVE_FIGURES set to True: figures will be saved automatically.
+FIGURE_TYPE = ".png"
+# FIGURE_TYPE can be any format matplotlib accepts for saving figures, eg.
+# ".png", ".svg", ".pdf" etc.
+FIGURE_DIR = "./"
```

Comment:
FIGURE_DIR is now double defined. Please delete this one.

Reply:
oops...